src/main/java/HTTPServer.java

```diff
@@ -0,0 +1,32 @@
+import java.io.*;
+import java.net.*;
+import java.util.HashMap;
+
+
+public class HTTPServer {
```

Comment:
Consider changing this class name to something else (ex: Main). At a glance, one would erroneously assume that HTTPServer is a subclass of Server given their names.

Reply:
I've changed the name to `Main`.